### ISSUE TITLE
[Feature Request] Save lr_finder plot instead of its showing

### ISSUE THREAD

**Issue by ahkarami** [pytorch/ignite#1926]
It would be great, if one can save plot of LRFinder module instead of showing it. Because, for example, in many cases we run our training scripts on Servers (e.g., AWS) and it is better to save the result plot (loss against learning rate plot) instead of showing it.
Thanks

**Comment by vfdev-5** [comment]
@ahkarami thanks for the suggestion ! I agree that [plot method](https://github.com/pytorch/ignite/blob/13a42b90589112e67bcc6e1780e7c826a5a34199/ignite/contrib/handlers/lr_finder.py#L189) could return the figure and have filepath optional argument to save current plot on disk.
